.pre-commit-config.yaml

```diff
@@ -327,7 +327,16 @@ repos:
           The word(s) should be in lowercase." && exec codespell "$@"' --
         language: python
         types: [text]
-        exclude: material-icons\.css$|^images/.*$|^RELEASE_NOTES\.txt$|^.*package-lock\.json$|^.*/kinglear\.txt$|^.*pnpm-lock\.yaml$|.*/dist/.*
+        exclude: >
+          (?x)
+          material-icons\.css$|
+          ^images/.*$|
+          ^RELEASE_NOTES\.txt$|
+          ^.*package-lock\.json$|
+          ^.*/kinglear\.txt$|
+          ^.*pnpm-lock\.yaml$|
+          .*/dist/.*|
+          ^airflow-core/src/airflow/ui/src/i18n/locales/de/
```

Comment:
Oh we should probably exclude `locales/zh_TW` too?

Reply:
As long as it does not raise an error we can keep the list small :-D